### PR TITLE
Update user object on page initialization

### LINKED ISSUE #628: Update user object on page initialization
If user is logged in, then send an API call to retrieve a fresh user data and sites.

Should be done in `HeaderController` since it is used there.

### PR DESCRIPTION
**Pull request by ArsalanSavand**
# Bug fixes
Fixes #628

### CODE REVIEW

File: app/header/header_controller.js [hunk 1]
```diff
@@ -138,6 +151,7 @@ function HeaderController($scope, $rootScope, $state, $stateParams,
   $scope.$on("gonevisDash.UserController:update", constructor);
 
   constructor();
+  $scope.refreshData();
```
Comment (Alir3z4) [inline app/header/header_controller.js]: So is this happens only once when user open the browser or it always happens each time user navigates to other pages ?

**Comment by Alir3z4** [comment]
is this happens only once when user open the browser or it always happens each time user navigates to other pages ?

**Comment by AmirSavand** [comment]
@Alir3z4 yes, the controller is initiated upon page load.

**Comment by ArsalanSavand** [comment]
Blocked by backend

**Comment by Alir3z4** [comment]
Alright, GN-513 has been labeled for this.
Once implemented this issue will be updated ;)

**Comment by Alir3z4** [comment]
@ArsalanSavand this is not blocked anymore
Use JWT Refresh endpoint with the current JWT token:

`/api/v1/account/refresh/`

**Comment by ArsalanSavand** [comment]
Done.

**Pull request merged**